test/performance/range/views/view_drop_view_take_benchmark.cpp

```diff
@@ -62,34 +62,35 @@ void sequential_read(benchmark::State & state)
+inline constexpr auto seqan3_views_take = seqan3::detail::take_fn<false, false>{};
 
 BENCHMARK_TEMPLATE(sequential_read, std::string,         void,                              void);
 BENCHMARK_TEMPLATE(sequential_read, std::string,         decltype(std::views::drop), decltype(std::views::take));
-BENCHMARK_TEMPLATE(sequential_read, std::string,         decltype(seqan3_views_drop),      decltype(seqan3::views::take));
+BENCHMARK_TEMPLATE(sequential_read, std::string,         decltype(seqan3_views_drop),      decltype(seqan3_views_take));
 
 BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, void,                              void);
 BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, decltype(std::views::drop), decltype(std::views::take));
-BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, decltype(seqan3_views_drop),      decltype(seqan3::views::take));
+BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, decltype(seqan3_views_drop),      decltype(seqan3_views_take));
 
 BENCHMARK_TEMPLATE(sequential_read, std::deque<uint8_t>,  void,                              void);
 BENCHMARK_TEMPLATE(sequential_read, std::deque<uint8_t>,  decltype(std::views::drop), decltype(std::views::take));
-BENCHMARK_TEMPLATE(sequential_read, std::deque<uint8_t>,  decltype(seqan3_views_drop),      decltype(seqan3::views::take));
+BENCHMARK_TEMPLATE(sequential_read, std::deque<uint8_t>,  decltype(seqan3_views_drop),      decltype(seqan3_views_take));
 
 BENCHMARK_TEMPLATE(sequential_read, std::list<uint8_t>,   void,                              void);
 BENCHMARK_TEMPLATE(sequential_read, std::list<uint8_t>,   decltype(std::views::drop), decltype(std::views::take));
-BENCHMARK_TEMPLATE(sequential_read, std::list<uint8_t>,   decltype(seqan3_views_drop),      decltype(seqan3::views::take));
+BENCHMARK_TEMPLATE(sequential_read, std::list<uint8_t>,   decltype(seqan3_views_drop),      decltype(seqan3_views_take));
 
 BENCHMARK_TEMPLATE(sequential_read, std::forward_list<uint8_t>,   void,                              void);
 BENCHMARK_TEMPLATE(sequential_read, std::forward_list<uint8_t>,   decltype(std::views::drop), decltype(std::views::take));
-BENCHMARK_TEMPLATE(sequential_read, std::forward_list<uint8_t>,   decltype(seqan3_views_drop),      decltype(seqan3::views::take));
+BENCHMARK_TEMPLATE(sequential_read, std::forward_list<uint8_t>,   decltype(seqan3_views_drop),      decltype(seqan3_views_take));
 
 BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, void,                              void,                              true);
 BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, decltype(std::views::drop), decltype(std::views::take), true);
-BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, decltype(seqan3_views_drop),      decltype(seqan3::views::take),      true);
+BENCHMARK_TEMPLATE(sequential_read, std::vector<uint8_t>, decltype(seqan3_views_drop),      decltype(seqan3_views_take),      true);
```

Comment:
those are all unaligned, should I correct all of them?

Reply:
Oh yes I missed that. 
Decide for yourself, if it is not too much effort, then do it. But I think it is not so important.

Reply:
I saw that too and decided to don't do this, because the PR would have become unreadable.